stratum/hal/lib/tdi/tdi_sde_table_entry.cc

```diff
@@ -69,6 +69,9 @@ ::util::Status TdiSdeWrapper::InsertTableEntry(
   if (status == BF_ALREADY_EXISTS) {
     return MAKE_ERROR(::util::error::Code::ALREADY_EXISTS)
            << "Duplicate table entry with " << dump_args();
+  } else if (status == BF_NO_SPACE) {
+    return MAKE_ERROR(::util::error::Code::RESOURCE_EXHAUSTED)
+           << "Table is full, resource exhausted " << dump_args();
```

Comment:
```suggestion
           << "Table is full " << dump_args();
```
"Table is full" expresses the full nature of the error. "resource exhausted" is redundant.